pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,183 @@
+# Running Buildkite Agent on Google Cloud Platform
+
+The Buildkite Agent can be run on [Google Cloud Platform](https://cloud.google.com). You may like to run the agent using individual VM instances on Google Compute Engine for fine control over long–lived agents. Or you may like to run a scalable cluster of agents using Docker and Kubernetes on the Google Container Engine to perform Docker–based builds.
+
+<%= toc %>
+
+## Running the agent on Google Compute Engine
+
+To run the agent on your own [Google Compute Engine](https://cloud.google.com/compute) instance use whichever installer matches your instance type. For example, launch an instance using the [latest Ubuntu LTS image](https://console.cloud.google.com/launcher?cat=OS&q=ubuntu) and follow [our Ubuntu setup instructions](/docs/agent/ubuntu).
```

Comment:
This format is kind of hard to comment on little bits of words! 😄 a bunch of small suggestions in one:

> To set up the agent on your own Google Compute Engine instance, run the Buildkite installer for your instance type. A full list of available installers can be found in the sidebar. For example:
> 1. Launch your instance using the latest Ubuntu LTS image
> 2. Follow our Ubuntu setup instructions.